spectroscopy/code_src/sdss_functions.py

```diff
@@ -39,7 +40,7 @@ def SDSS_get_spec(sample_table, search_radius_arcsec, data_release):
         try:
             xid = SDSS.query_region(search_coords, radius=search_radius_arcsec
                                     * u.arcsec, spectro=True, data_release=data_release)
-        except requests.HTTPError:
+        except (requests.HTTPError, Timeout):
             print(f"Encountered SDSS service error when querying region for source {stab['label']}. Skipping.")
```

Comment:
while you are at it, could you also change these to be warnings rather than prints? I believe we did that for other functions already.

Reply:
done.